Sources/Parley/Data/Collections/MessageCollection.swift

```diff
@@ -1,24 +1,24 @@
 import Foundation
 
-internal struct MessageCollection: Codable, Equatable {
```

Comment:
I've remove the `internal` access modifier because that is the default and it was not used everywhere;